drivers/docker/driver.go

```diff
@@ -612,7 +612,18 @@ func (d *Driver) loadImage(task *drivers.TaskConfig, driverConfig *TaskConfig, c
 }
 
 func (d *Driver) containerBinds(task *drivers.TaskConfig, driverConfig *TaskConfig) ([]string, error) {
-
+	if runtime.GOOS == "windows" {
+		imageConfig, _ := client.InspectImage(driverConfig.Image)
```

Comment:
I would have expected `containerBinds` to be a pure function that doesn't mutate its inputs.  May I suggest introducing `convertAllocPathsForWindowsLCow` like function instead?

Also, please handle the error condition where `imageConfig` is nil.

Reply:
I have moved the variable change to a function, which also handles the errors returned by InspectImage.